azure-pipelines.yml

```diff
@@ -1,7 +1,9 @@
 pool:
   vmImage: 'Ubuntu 16.04'
 
-container: dtzar/conda3mlops:latest
+container:
+  image: mlopscr.azurecr.io/public/mlops/mlopspython:latest

```

Comment:
Can't we use the public image?

Reply:
Not yet. Until it is published on Docker Hub we are using this one.